src/lib/embed.js

```diff
@@ -181,11 +181,10 @@ export function resizeEmbeds(parent = document) {
                 continue;
             }
 
+            // Change padding-bottom of the enclosing div to accommodate
+            // card carousel without distorting aspect ratio
             const space = iframes[i].parentElement;
```

Comment:
should we check if the parent div has a paddingBottom or paddingTop before making adjustments?

Reply:
parent div will have a paddingTop by default based on the aspect ratio. 
Is there a case where a user may add custom paddingBottom to the div?